config/prow/config_knative.yaml

```diff
@@ -471,6 +485,20 @@ periodics:
       dot-dev: true
       go113: true
 
+  knative/client-contrib:
+    - continuous: true
+      dot-dev: true
+      go113: true
+    - nightly: true
```

Comment:
You haven't added `./hack/release.sh` in that repo, so these jobs (nightly, dot-release, auto-release)  are not needed.

Reply:
good point. I will remove it for now and add later if needed.